app/src/main/java/com/github/se/eventradar/model/repository/user/FirebaseUserRepository.kt

```diff
@@ -136,28 +136,37 @@ class FirebaseUserRepository(db: FirebaseFirestore = Firebase.firestore) : IUser
       Resource.Failure(e)
     }
   }
-}
 
-private fun getMaps(user: User): Pair<Map<String, Any?>, Map<String, Any?>> {
-  val privateMap =
-      mutableMapOf(
-          "firstName" to user.firstName,
-          "lastName" to user.lastName,
-          "phoneNumber" to user.phoneNumber,
-          "birthDate" to user.birthDate,
-          "email" to user.email,
-      )
-
-  val publicMap =
-      mutableMapOf(
-          "profilePicUrl" to user.profilePicUrl,
-          "qrCodeUrl" to user.qrCodeUrl,
-          "username" to user.username,
-          "accountStatus" to user.accountStatus,
-          "eventsAttendeeList" to user.eventsAttendeeSet,
-          "eventsHostList" to user.eventsHostSet,
-          "friendsList" to user.friendsSet,
-      )
-
-  return Pair(publicMap, privateMap)
+  private fun getMaps(user: User): Pair<Map<String, Any?>, Map<String, Any?>> {
+    val privateMap =
+        mutableMapOf(
+            "firstName" to user.firstName,
+            "lastName" to user.lastName,
+            "phoneNumber" to user.phoneNumber,
+            "birthDate" to user.birthDate,
+            "email" to user.email,
+        )
+
+    val publicMap =
+        mutableMapOf(
+            "profilePicUrl" to user.profilePicUrl,
+            "qrCodeUrl" to user.qrCodeUrl,
+            "username" to user.username,
+            "accountStatus" to user.accountStatus,
+            "eventsAttendeeList" to user.eventsAttendeeSet,
+            "eventsHostList" to user.eventsHostSet,
+            "friendsList" to user.friendsSet,
+        )
+
+    return Pair(publicMap, privateMap)
+  }
```

Comment:
sorry I didn't specify but this should not be in this PR, I had to copy and paste it from Nathan In order to test my code. once, pull from main once Nathan has pushed, It will be there